tasks/source.ps1

```diff
@@ -0,0 +1,54 @@
+[CmdletBinding()]
+Param(
+  [string]$command = '',
```

Comment:
Is setting these to empty strings actually what we want to do here?

Reply:
I think relates back to determining whether or not a specific value was passed as in the install script above.

Reply:
Note also that you can limit the list of possible values via `ValidateSet`.